data/resources/morale affected.txt

```diff
@@ -0,0 +1,137 @@
+# Copyright (c) 2021 by Luke Arndt
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+# This file is for configuring how various events affect morale.
+
+# All fields are optional except "morale affected" and "unique id".
+#
+# Record format:
+# "morale affected" "unique id"
+#   "amount divided by crew members" (number; divided by the number of crew on the ship)
+#   "amount per crew member" (number; multiplied by the number of crew on the ship)
+#   "flat amount" (number; a flat morale change, unaffected by crew count)
+#   "fleet message" (string; displayed if the event applies to a fleet)
+#   "parked multiplier" (number; the morale change is multiplied by this for parked ships)
+#   "ship message" (string; displayed if the event applies to a single ship)
+
+
+# Crew members have died on a ship in the fleet
+# This happens when a ship is destroyed and when crew are lost during a capture attempt
+# Multiplied by the number of crew members that died
+
+"morale affected" "death in fleet"
+  "amount divided by crew members" -1
+  "flat amount" -0.1
+  "fleet message" "In response to crew member deaths,"
+
```

Comment:
I think there should be another step of removal here.  Just as witnessing a death is more traumatising than being in a battle where someone dies elsewhere, being completely out of the picture when it happens is easier to deal with.
```suggestion
  "parked multiplier" 0.5

```